db/organizations/connectwave.eno

```diff
@@ -0,0 +1,9 @@
+name: Connectwave
+website_url: https://www.danawa.com/
+privacy_policy_url: https://www.danawa.com/info/helprule_private.html
+privacy_contact: privacy@cowave.kr
+country: KR
+description: "Danawa is a price comparison site like BestBuy."
```

Comment:
I found the following description. Do you think it is factually correct and makes sense to change to it as the company description?

```
ConnectWave Co Ltd, formerly DANAWA Co., Ltd, is a Korea-based company mainly engaged in the operation of online price comparison websites.
```

Reply:
It seems like more descriptive than my one. I had no idea how to explain the website. :)